src/components/PopoverListItem/PopoverListItem.module.css

```diff
@@ -0,0 +1,32 @@
+/*------------------------------------*\
+    # POPOVER LIST ITEM
+\*------------------------------------*/
+
+/**
+ * PopoverListItem
+ */
+.popover-list-item {
+  display: flex;
+  font-size: var(--eds-font-size-14);
+  padding: var(--eds-size-1) var(--eds-size-2);
+  background-color: var(--eds-theme-color-form-background);
+  cursor: pointer;
+}
+
+.popover-list-item--active {
+  background-color: var(--eds-theme-color-background-neutral-default-hover);
+}
+
+.popover-list-item--disabled {
+  color: var(--eds-theme-color-text-disabled);
+  pointer-events: none;
+}
+
+.popover-list-item__icon {
+  padding-right: var(--eds-size-1-and-half);
+}
+
+/* right padding applies space for the icon and the padding for that icon container */
+.popover-list-item__no-icon {
+  padding-right: calc(var(--eds-size-3) + var(--eds-size-1-and-half));
```

Comment:
Are you adding them together because there's no `4-and-a-half`? I would just pass it in in `rem`s.. We don't use the `size` tokens everywhere. (I'm still not convinced size tokens like these are useful.)

Reply:
I use the two values to show where the quantity comes from. Just above (L26) we use 1.5 for padding, and here the icon size happens to be 3. Lots of components are using various sizes so didn't think they were not part of the existing patterns.

This is fairly complex too. The sizes have a relationship to one another, based on some base size and multipliers ... having tokens for each one means defining different sizes is more work than it has to be IMO. Something to discuss (moving to defining a base grid value and using `calc()` or similar to get other quantities)

Reply:
>I use the two values to show where the quantity comes from. Just above (L26) we use 1.5 for padding, and here the icon size happens to be 3.

Ah, cool. Could you move the comment from line 29 down one step so it's above this styling?